### DISCUSSION TITLE
[gatsby-plugin-image] StaticImage wont work with emotion styledComponent

### DISCUSSION THREAD

**Discussion by ahmadkhalaf1** [gatsbyjs/gatsby#30164] [Help]
Hello , i am trying to use StaticImage with styledComponent but the image wont show up , 
this is my code 
```
const StlyedImg = styled(StaticImage)`
  position: absolute;
  top: -4rem;
  width: 200px;
  height: 140px;
  margin: 0 0 0 -7.5rem;
  padding: 0;
  display: block;
  left: 50%;
  transform: rotate(180deg);
  overflow: unset;
`;
```
```
<StlyedImg src="../images/flower_06_reminder.png" alt="flower" width={200} height={140} />
```

however loading the image outside of styledComponent will render just fine.

**Comment by kelvindecosta** [comment]
I had the same issue, so I just used `GatsbyImage` instead 🤷🏾‍♂️

**Reply by ascorbic** [reply]
It won't work because when we extract static images at build time, it doesn't recognise that `StyledImg` is a `StaticImage` component. Babel needs to be able to statically-analyse the props, which I can't do if they're inside a HOC like `styled()`. Using the css prop on the component should work though, or just use a styled wrapper.

**Reply by kelvindecosta** [reply]
Ah, I knew there was something to do with ensuring that the component was indeed a `StaticImage`.
It didn't occur to me to use the `css` prop.

Thanks for the explanation and solution!
